src/kixi/nhs/board_report.clj

```diff
@@ -116,7 +222,7 @@
          data        (storage/get-resource-data ckan-client resource_id)]
     (patient-experience-of-gp-services recipe data)))
 
-(defn process-patient-experience-recipes [ckan-client recipes]
+(defn patient-experience-ethnicity-analysis [ckan-client recipes]
```

Comment:
OK, split indicator: 212 and indicators: 213, 214, 215, 216, 217 into separate patient-experience/deprivation.clj and patient-experience/ethnicity as discussed and update the comments.